code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -270,8 +270,9 @@
 
 /datum/species/proc/breathe(mob/living/carbon/human/H)
 	var/datum/organ/lungs/lung = H.get_int_organ_datum(ORGAN_DATUM_LUNGS)
-	if(HAS_TRAIT(H, TRAIT_NOBREATH) && lung)
-		lung.clear_alerts(H)
+	if(HAS_TRAIT(H, TRAIT_NOBREATH))
+		if(lung)
+			lung.clear_alerts(H)
```

Comment:
```suggestion
		lung?.clear_alerts(H)
```

Reply:
This causes runtimes with machine people

Reply:
oh wait im dumb
sorry

Reply:
done, missed the `?` that caused runtimes.